_posts/2022-11-03-call-for-proposals-for-djangocon-2023-website.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title:  "Call for Proposals for DjangoCon US 2023 Website!"
+date: 2022-11-03 09:00:00
+categories: ""
+author_name : DEFNA
+author_url : /about
+author_avatar: defna
+show_avatar : true
+read_time : ""
+feature_image: 2022-11-03
+show_related_posts: false
+permalink: /announcements/2022/11/03/call-for-proposals-for-djangocon-2023-website/
+---
+
+### Introduction
+
+We are looking for assistance with our DjangoCon US website design, and conference swag and art direction. 
+
+Our main goal for 2023 [(repo here)](https://github.com/djangocon/2023.djangocon.us) is to update our color scheme, logos, and [style guide](https://2022.djangocon.us/styleguide/) to fit the theme of the new city.
+
+We have all of our 2022 data to work with which includes all of our content along with our speakers and schedule (talks, tutorials, and sprint information) which can be used to verify and test with.
+
+We'd also like your help with our swag and art direction which includes our backdrops during the conference, videos, signage, along with special swag which we give out at the conference.
+
+## Website Overview
+
+In 2017, the DjangoCon US website was re-built using the static site generator Jekyll and hosted on GitHub. There are a number of advantages to this approach:
```

Comment:
We should make it clear that we're not wedded to that approach (Jekyll), but we do want it to be clear that we want contributors not to be required to know how to write even HTML to be able to add content.

Reply:
Maybe I should put back the Django reference that we had before. I took it out because it seemed beside the point five years later.

Reply:
I've updated the text: https://github.com/DEFNA/defna.org/pull/37/commits/87294dfa86a4c6b988fb43930ae49049b175b4f6